src/proxy/index.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable max-len */
 const proxyApp = require('express')();
 const bodyParser = require('body-parser');
-const routes = require('./routes');
+const routes = require('./routes').router;
```

Comment:
@sammoorhouse, can you explain why this change is needed?

Reply:
Because I expose {router, handleMessage} from proxy/routes/index.js